bsp/qemuarma15/qemuarma15-standard.scc

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: MIT
+define KMACHINE qemuarma15
+define KTYPE standard
+define KARCH arm
+
+include ktypes/standard/standard.scc
+# we aren't carrying patches specific to this platform, so let's
+# skip the branch for now.
+# branch qemuarma15
+
+include features/bluetooth/bluetooth.scc
+include features/bluetooth/bluetooth-usb.scc
+include features/bluetooth/bluetooth-uart.scc
+include features/ieee802154/ieee802154.scc
+include features/mac802154/mac802154.scc
+
+include cfg/virtio.scc
+include cfg/dmaengine.scc
+include cfg/timer/rtc.scc
```

Comment:
Should we align this fine with our the other bsps we have? Most of the common include bits are generic.